postgres/datadog_checks/postgres/postgres.py

```diff
@@ -318,10 +319,17 @@ def dynamic_queries(self):
                 ]
             )
 
-        if self.version < V10:
-            queries.append(QUERY_PG_CONTROL_CHECKPOINT_LT_10)
+        if self.wal_level == 'logical':
+            self.log.debug("wal_level is logical, adding control checkpoint metrics")
+
+            if self.version < V10:
```

Comment:
Can you swap this to be `>= V10` so the common case comes first rather than in the `else`?